VTEX - Logistics API.json

```diff
@@ -3224,102 +3224,92 @@
-                                                        "format": "int32"
+                                                        "format": "int32",
+                                                        "description": "Total quantity of items of the given SKU."
                                                     },
                                                     "reservedQuantity": {
                                                         "type": "integer",
-                                                        "format": "int32"
+                                                        "format": "int32",
+                                                        "description": "SKU reserved quantity."
                                                     },
                                                     "hasUnlimitedQuantity": {
-                                                        "type": "boolean"
+                                                        "type": "boolean",
+                                                        "description": "When set as `false`, every sold unit decreases the item's available quantity. When set as `true`, the SKU from the warehouse is always available for sale. No matter how many units are sold, the SKU available quantity never decreases, and the SKU is never out of stock."
+                                                    },
+                                                    "timeToRefill": {
+                                                        "type": "string",
+                                                        "description": "Time to refill.",
+                                                        "nullable": true
+                                                    },
+                                                    "dateOfSupplyUtc": {
+                                                        "type": "string",
+                                                        "description": "Supply date and time in [ISO 8601 time zone offset format](https://learn.microsoft.com/en-us/rest/api/storageservices/formatting-datetime-values), as in `YYYY-MM-DDThh:mm:ss.ssZ`.",
+                                                        "nullable": true
+                                                    },
+                                                    "leadTime": {
+                                                        "type": "string",
+                                                        "description": "[Lead time](https://help.vtex.com/en/tutorial/lead_time-shipping-time-at-sku-level--16yv5Mkj6bTyWR1hCN2f4B) is an optional time configuration you can make for a SKU in a warehouse. It can be handling time, fabrication or how long it takes for the item to be available to be shipped to customers. The lead time is a part of the total shipping time and will be considered for shipping date calculation. The format is `DD.HH:MM:SS`."
```

Comment:
```suggestion
                                                        "description": "[Lead time](https://help.vtex.com/en/tutorial/lead-time-shipping-time-at-sku-level--16yv5Mkj6bTyWR1hCN2f4B) is an optional time configuration you can make for a SKU in a warehouse. It can be handling time, fabrication or how long it takes for the item to be available to be shipped to customers. The lead time is a part of the total shipping time and will be considered for shipping date calculation. The format is `DD.HH:MM:SS`."
```